configure.ac

```diff
@@ -34,6 +34,14 @@ dnl Checks for programs
 AC_PROG_CC
 AM_PROG_CC_C_O
 
+AC_ARG_WITH([libfabric],
+            AC_HELP_STRING([--with-libfabric], [Use non-default libfabric location - default NO]),
+            [AS_IF([test -d $withval/lib64], [fab_libdir="lib64"], [fab_libdir="lib"])
+             [CPPFLAGS="$CPPFLAGS -I $withval/include"]
+             [LIBS="$LIBS -lfabric"]
+             [LDFLAGS="-L$withval/$fab_libdir $LDFLAGS"]],
```

Comment:
Stylistically, it's *correct* but *weird* to surround each of lines 40-42 with [ and ] -- but then not put commas between them, so they're all still technically one field.  Instead, I'd do:

```shell
            [AS_IF([test -d $withval/lib64], [fab_libdir="lib64"], [fab_libdir="lib"])
             CPPFLAGS="$CPPFLAGS -I $withval/include"
             LDFLAGS="-L$withval/$fab_libdir $LDFLAGS"],
            [])
```

I.e., the CPPFLAGS and LDFLAGS are all part of the same m4 block as the AS_IF.